daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/UnseparatedSequenceChildParseResultHelper.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.daffodil.processors.parsers
+
+import org.apache.daffodil.processors.Success
+import org.apache.daffodil.util.Maybe
+import org.apache.daffodil.processors.ElementRuntimeData
+import org.apache.daffodil.exceptions.Assert
+import org.apache.daffodil.schema.annotation.props.EmptyElementPolicy
+import org.apache.daffodil.processors.ModelGroupRuntimeData
+
+trait UnseparatedSequenceChildParseResultHelper
```

Comment:
Note to reviewers: I am no longer certain these helpers couldn't be merged into the parser clases, so that we don't have two distinct object taxonomies. However, I am disinclined to do that merge until this code has proven itself robust over time, and all the behavior has been proven correct for the schemas we need to work (including EDIFACT which doesn't work as of this comment writing.) Eventually these could be merged which would simplify this mixin-intensive system for isolating exactly where different behaviors are really needed.

Reply:
Add comment to code to this effect.